src/css/styles.css

```diff
@@ -0,0 +1,67 @@
+/*
+ * react-circular-progressbar styles
+ * All of the styles in this file are configurable!
+ */
+
```

Comment:
- [ ] I noticed that you have commented on codes. I would advise you to look at it and remove those which are not necessary. This is because you need to be having clean code, a way to show professionalism.